tests/backends/base/test_creation.py

```diff
@@ -40,3 +41,29 @@ def test_custom_test_name_with_test_prefix(self):
         test_connection.settings_dict['TEST'] = {'NAME': test_name}
         signature = BaseDatabaseCreation(test_connection).test_db_signature()
         self.assertEqual(signature[3], test_name)
+
+
+@mock.patch.object(connection, 'ensure_connection', return_value=None)
+@mock.patch('django.core.management.commands.migrate.Command.handle', return_value=None)
+class TestDbCreationTests(SimpleTestCase):
+    def test_migrate_test_setting_false(self, mocked_migrate, mocked_ensure_connection):
+        creation = connection.creation_class(connection)
+        try:
+            saved_settings = connection.settings_dict
+            connection.settings_dict['TEST']['MIGRATE'] = False
+            with mock.patch.object(creation, '_create_test_db', return_value=None):
```

Comment:
Same here, which appears in each test method.

```suggestion
            with mock.patch.object(creation, '_create_test_db'):
```